vm/ieleClientUtils.ml

```diff
@@ -2,26 +2,11 @@ open Yojson
 open Yojson.Basic.Util
 open Msg_types
 
-let file = Sys.argv.(1)
-
-let () = if Array.length Sys.argv <> 3 then
-  prerr_endline ("usage: " ^ Sys.argv.(0) ^ " <file.iele.json> <port>")
-
-let json = Yojson.Basic.from_channel (open_in file)
-
-let context = Secp256k1.Context.create [Secp256k1.Context.Sign; Secp256k1.Context.Verify]
 let hash () = Cryptokit.Hash.keccak 256
 
-let test =
-  match json with
-  `Assoc [(_, test)] -> test
-| _ -> failwith "Invalid json structure: expected an object at top level."
-
 let sort l =
```

Comment:
A more descriptive name like `sort_on_fst` may be appropriate now that this is a reused function.